packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -143,14 +144,15 @@ export function PanelChrome({
   onFocus,
   onMouseMove,
   onMouseEnter,
+  onDragStart,
   showMenuAlways = false,
 }: PanelChromeProps) {
   const theme = useTheme2();
   const styles = useStyles2(getStyles);
   const panelContentId = useId();
   const panelTitleId = useId().replace(/:/g, '_');
   const { isSelected, onSelect, isSelectable } = useElementSelection(selectionId);
-  const pointerDownPos = useRef<{ screenX: number; screenY: number }>({ screenX: 0, screenY: 0 });
+  const pointerDownEvt = useRef<React.PointerEvent | undefined>();
```

Comment:
Do we need these changes? Yesterday we merged a fix for differentiating between drag start and selection.